frontend/pages/inventory.tsx

```diff
@@ -1,9 +1,147 @@
-import React from 'react';
+import { FC, ReactNode, useState, useEffect } from 'react';
+import { API_URL } from '../config';
+import { toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 
-function Inventory() {
+interface InventoryLayoutProps {
+    children: ReactNode;
+}
+
+const InventoryLayout: FC<InventoryLayoutProps> = ({ children }) => {
     return (
-        <h1>Inventory</h1>
+        <div className="container mx-auto mt-8">
+            <h1 className="text-4xl font-bold mb-4">Inventory</h1>
+            <div className="overflow-x-auto">{children}</div>
+        </div>
     );
+};
+
+interface InventoryItem {
+    id: number;
+    name: string;
+    description: string;
+    quantity: number;
+}
```

Comment:
move this a specific file

Reply:
in component then ? cause it's not a page

Reply:
more of a `frontend/interfaces/Inventory.tsx`
@lucas-debize @LaolouRavon-Laforest where di you put yours ?